lang/fr-fr/lang.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+return [
+    'plugin' => [
+        'name' => 'JWTAuth',
+        'description' => 'Authentification par Token Web JSON.',
+    ],
+    'permissions' => [
+        'settings' => 'Gestion de JWTAuth',
+    ],
+    'settings' => [
+        'menu_label' => 'JWTAuth',
+        'menu_description' => 'Configuration de JWTAuth',
+        'tabs' => [
+            'urls' => 'Options d''URL',
```

Comment:
I don't think that using the double `single quotes` here will work. The PHP string is already using `single quotes`, which would be better if you need to use the single quote character to the language if you replace the quotes that limit the string with double-quotes.

Reply:
For example, something like this:

```'urls' => "Options d''URL", ```